impeller/docs/faq.md

```diff
@@ -100,3 +100,113 @@
     as development progresses.
 * How do you run `impeller_unittests` with Playgrounds enabled?
   * Specify the `--enable_playground` command-line option.
+* Describe Impeller in a Tweet.
+  * "Impeller is AOT mode for rendering."
+* Why did the Flutter team build Impeller?
+  * The short answer, for consistent performance. The longer answer...
+  * Flutter applications, via their use of Skia, were susceptible to the problem
+    of jank due to shader compilation. Such problems were [reported as early as
+    2015](https://github.com/flutter/flutter/issues/813).
+  * Shaders are tiny programs that run on the GPU when a specific combination of
+    rendering intent is requested by the application. Flutters API is extremely
+    expressive and it was thought to be impossible to determine the entire set
+    of shaders an application might need at runtime. So, applications would
+    create and compile these shaders just-in-time as they were needed. Once they
+    were compiled, they could be cached.
+  * But this led to the problem of jank during the initial run of the
```

Comment:
maybe: during the initial run of performance sensitive areas of the app

otherwise, it might sound like we mean "a bunch of jank when the app starts up", which I don't think we quite mean here.

Reply:
Done.